packages/@uppy/companion/src/server/provider/onedrive/index.js

```diff
@@ -137,7 +137,7 @@ class OneDrive extends Provider {
   _error (err, resp) {
     if (resp) {
       const fallbackMsg = `request to ${this.authProvider} returned ${resp.statusCode}`
-      const errMsg = resp.body.error ? resp.body.error.message : fallbackMsg
+      const errMsg = (resp.body || {}).error ? resp.body.error.message : fallbackMsg
```

Comment:
this change can be ignored. I accidentally checked out of [this branch](https://github.com/transloadit/uppy/pull/2264/files)